Schema/sim.simXml.schema.json

```
@@ -0,0 +1,35 @@
+{
+    "$id": "https://vng.opencatalogi.nl/schemas/sim.simXml.schema.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "version": "0.1",
+    "title": "SimXml",
+    "type": "object",
+    "properties": {
+      "stuurgegevens": {
+        "$ref": "#/definitions/Stuurgegevens"
+      },
+      "body": {
+        "$ref": "#/definitions/Body"
+      },
+      "bijlagen": {
+        "$ref": "#/definitions/Bijlage"
+      },
```

bijlagen is an array in stoplight.

```suggestion
      "bijlagen": {
         "type": "array",
         "items": {
               "$ref": "#/definitions/Bijlage"
           }
      },
```